src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -1162,6 +1160,21 @@ InferenceProfiler::SummarizeLatency(
   return cb::Error::Success;
 }
 
+std::tuple<uint64_t, double>
+InferenceProfiler::GetTotalLatencies(const std::vector<uint64_t>& latencies)
+{
+  uint64_t tol_latency_ns{0};
+  double tol_square_latency_us{0.0};
+
+  for (const auto& latency : latencies) {
+    tol_latency_ns += latency;
+    tol_square_latency_us +=
+        (latency * static_cast<double>(latency)) / (1000 * 1000);
```

Comment:
I think by converting uint64 to double we might lose some of the precision. How about using the alternative formula for calculating variance which finds the squared difference from mean so that the numbers are smaller and can prevent overflow?

Reply:
Changed calculation to use difference from mean to avoid overflow 👍